watchface

```diff
@@ -0,0 +1,406 @@
+#! /bin/bash
+VERSION="1.0"
+
+function showVersion {
+    echo "watchface v${VERSION}"
+}
+
+function showHelp {
+    showVersion
+    cat << EOF
+watchface [option] [command...]
+Utility functions for AsteroidOS watchfaces.  By default, uses "Developer Mode"
+over ssh, but can also use "ADB Mode" using ADB.
+
+Available options:
+-h or --help    prints this help screen and quits
+-a or --adb     uses ADB command to communicate with watch
+-b or --boot    reboot watch after deploying multiple watchfaces
+-g or --gui     use the GTK+ gui
+-p or --port    specifies a port to use for ssh and scp commands
+-r or --remote  specifies the remote (watch)  name or address for ssh and scp commands
+-q or --qemu    communicates with QEMU emulated watch (same as -r localhost -p 2222 )
+-w or --wall WP sets the wallpaper for deploy or test to the named file
+
+Available commands:
+version         displays the version of this program and exits
+deploy WF       pushes the named watchface to the watch and activates it
+clone WF NEWWF  clones the named watchface WF to new watchface NEWWF
+test WF         sets the active watchface to the named watchface
```

Comment:
I think you meant to write another description here ?

Reply:
Right!  Fixed.